packages/fetchai/skills/weather_station/handlers.py

```diff
@@ -285,11 +274,8 @@ def _handle_inform(self, msg: FIPAMessage, dialogue: Dialogue) -> None:
             proposal = cast(Description, dialogue.proposal)
             ledger_id = cast(str, proposal.values.get("ledger_id"))
             is_settled = self.context.ledger_apis.is_tx_settled(ledger_id, tx_digest)
-            is_valid = self.context.ledger_apis.validate_transaction(
```

Comment:
we should discuss why this has been removed here

Reply:
I just removed it for now just to be easier for review. Once I have the correct function validate_transaction() I ll update the packages to use it.